packages/syft/src/syft/lib/sympc/rst_share.py

```diff
@@ -0,0 +1,55 @@
+# third party
+from sympc.tensor import ReplicatedSharedTensor
+
+# syft relative
+from ...generate_wrapper import GenerateWrapper
+from ...lib.torch.tensor_util import protobuf_tensor_deserializer
+from ...lib.torch.tensor_util import protobuf_tensor_serializer
+from ...proto.lib.sympc.replicatedshared_tensor_pb2 import (
+    ReplicatedSharedTensor as ReplicatedSharedTensor_PB,
+)
+from .session_util import protobuf_session_deserializer
+from .session_util import protobuf_session_serializer
+
+
+def object2proto(obj: object) -> ReplicatedSharedTensor_PB:
+    share: ReplicatedSharedTensor = obj
+
+    session = protobuf_session_serializer(share.session)
+    proto = ReplicatedSharedTensor_PB(session=session)
+
+    tensor_proto_list = []
+
+    for tensor in share.shares:
+        tensor_proto_list.append(protobuf_tensor_serializer(tensor))
+
+    if len(tensor_proto_list) != 0:
```

Comment:
MAYBE : We could do directly , `proto.tensor.extend(tensor_proto_list)`

Reply:
Yep, it is possible. Thanks.